allennlp/models/simple_tagger.py

```diff
@@ -120,11 +120,10 @@ def tag(self, text_field: TextField) -> Dict[str, Any]:
             An array of shape (text_input_length, num_classes), where each row is a
             distribution over classes for a given token in the sentence.
         """
-        text_field.index(self.vocab)
-        padding_lengths = text_field.get_padding_lengths()
-        array_input = text_field.as_array(padding_lengths)
-        model_input = arrays_to_variables(array_input, add_batch_dimension=True)
-        output_dict = self.forward(tokens=model_input)
+        instance = Instance({'tokens': text_field})
```

Comment:
Disagree with this change - with the others, wrapping the fields in `Instance` makes it less verbose - here, we only have a single field, so we gain no advantage by wrapping it with Instance.